multiversx/mx-chain-go
# Fix value as string on response for /transaction/pool by sender

## Reasoning behind the pull request
- invalid value returned
![image](https://user-images.githubusercontent.com/34831323/220344424-672ed53d-eaf2-42aa-b1d5-7a5cf0462520.png)

  
## Proposed changes
- use value as string

## Testing procedure
- standard system test with some txs in pool and call `/transaction/pool?by-sender=erd1...&fields=value` and make sure value is a valid string, not as in the above picture

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
